### PR TITLE
IPAAS-314: Have an Online and Demo versions of the workflow handlers.

### PR DESCRIPTION
**Pull request by chirino**
The demo version move to the desired state after 2 seconds.

### CODE REVIEW

File: controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package com.redhat.ipaas.controllers;
+
+import com.redhat.ipaas.model.integration.Integration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+@ConditionalOnProperty(value = "controllers.integration.enabled", havingValue = "noop")
```
Comment (KurtStam) [inline controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java]: controllers.integration.enabled can have values of true/false/noop?

Comment (chirino) [inline controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java]: thats it.

Comment (KurtStam) [inline controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java]: So what's the difference between noop and false? Maybe you can add some javadoc, and a paragraph to the readme?

Comment (chirino) [inline controllers/src/main/java/com/redhat/ipaas/controllers/NoopHandler.java]: noop does not update integrations, false basically enabled demo mode.  it update integration to the desired status after 5 seconds.  Naming might not be great but it's what's easiest to use with the current templates.

**Pull request merged**